src/shared/util.js

```diff
@@ -22,7 +22,8 @@ const RenderingIntentFlag = {
   ANY: 0x01,
   DISPLAY: 0x02,
   PRINT: 0x04,
-  ANNOTATION_FORMS: 0x20,
+  ANNOTATIONS_FORMS: 0x10,
```

Comment:
Since we have quite a few rendering intents now, it might be a good time to add a small comment to this enumeration stating what the various rendering intents do. For example, the display/print ones are fairly obvious, but the oplist one less so, and the distinction between forms and storage is also perhaps not immediately clear just from their names.

Reply:
Yeah, I suppose that on its own this isn't terribly helpful.
My question here is if referencing the `WorkerTransport.getRenderingIntent`-method helps explain the idea behind these flags (better than comment would), since I'm a bit worried that trying to explain the flags on their own will be difficult to follow without context (I often struggle to explain things in a clear and concise way).

There's a possible follow-up to this, see a WIP patch at https://github.com/Snuffleupagus/pdf.js/compare/includeAnnotationStorage-fix-caching...Snuffleupagus:AnnotationMode, which may also clarify things further once the new `annotationMode` option is properly documented.

Reply:
I've tried to add some docs, and I really hope that it makes sense!
To simplify things, as mentioned before, I'm referring to methods/options in the API since looking at the bigger picture will hopefully be clearer.